src/os/bluestore/KernelDevice.cc

```diff
@@ -521,7 +521,11 @@ void KernelDevice::_aio_thread()
 		devname.c_str(),
 		path.c_str(),
 		r,
-		aio[i]->iocb.aio_lio_opcode,
+#if defined(HAVE_POSIXAIO)
```

Comment:
Does not build:

```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/os/bluestore/KernelDevice.cc:527:25: error: 'struct aio_t' has no member named 'aio'; did you mean 'aio_t'?
                 aio[i]->aio.iocb.aio_lio_opcode,
                         ^~~
                         aio_t
src/os/CMakeFiles/os.dir/build.make:782: recipe for target 'src/os/CMakeFiles/os.dir/bluestore/KernelDevice.cc.o' failed
make[3]: *** [src/os/CMakeFiles/os.dir/bluestore/KernelDevice.cc.o] Error 1
```

Reply:
@xiexingguo 
I noticed, and fixed.
But now the error is in some seastar test...

Reply:
@wjwithagen it's fixed. sorry for the inconvenience.